Plugin/Controller/Adminhtml/Integration/SavePlugin.php

```diff
@@ -192,10 +192,18 @@ private function sendIntegrationToExtend($integrationId, $storeId)
                 'magentoStoreUuid' => $integrationStore->getStoreUuid(),
                 'magentoStoreId' => $storeId,
                 'magentoConsumerKey' => $oauth->getKey(),
-                'storeDomain' => $this->scopeConfig->getValue(
-                    Store::XML_PATH_UNSECURE_BASE_URL,
-                    'store',
-                    $storeId
+                'extendStoreId' => $integrationStore->getExtendStoreUuid(),
+                'storeDomain' => rtrim(
+                    str_replace(
```

Comment:
this removes the `https://` and the `/` from the `storeDomain`

Reply:
What is the metadata builder expecting? I could check but it feels weird doing this with no corresponding metadata builder changes

Reply:
yeah not my fave either but the `MetaDataBuilder` is just sending the `storeId` not the `storeDomain`

Reply:
Then why do we give it the store domain? Or am I not reading it right?

Reply:
it's in the `data` object here which is just passed along